### DISCUSSION TITLE
Feature Request: Delete images by filename

### DISCUSSION THREAD

**Discussion by Uncanny5180** [simulot/immich-go#104] [Ideas]
Hello, just a thought, but I'd love to see a wildcard/regex  function to delete photos by name as needed or after an import perhaps. Personally, I save a lot of screenshots, and many of them have found their way into my library when I had no desire or intent on keeping them long term. I'd love to see a function like delete *screenshot.jpg

**Comment by simulot** [comment]
Nice one!

I have in mind adding a command for doing this kind of thing.
We can extend a bit the possibilities:
- date range
- camera maker name
- delete an album and its content

and more risky: delete the result a search query... 
delete -query="crap"

![image](https://github.com/simulot/immich-go/assets/5980377/c2cce41a-8865-415d-8d34-4fabc51ea12d)

Maybe not...

**Reply by Uncanny5180** [reply]
Yeah that would be great too. I have a ton of the similar stuff there that I wouldnt mind organizing/cleaning.
